package.json

```diff
@@ -33,7 +33,7 @@
     "axios": "^0.19.0",
     "body-parser": "^1.19.0",
     "buying-catalogue-components": "^1.1.56",
-    "buying-catalogue-library": "^1.0.5",
+    "buying-catalogue-library": "^1.0.6",
```

Comment:
what does this bring in?

Reply:
I don't know being honest. I just updated it to the latest since I was having some issues with getting the Unauthenticated page to show. But it was something else that was causing that to break.